routingpy/routers/valhalla.py

```diff
@@ -513,6 +505,7 @@ def matrix(
         avoid_locations: Optional[List[List[float]]] = None,
         avoid_polygons: Optional[List[List[List[float]]]] = None,
         units: Optional[str] = None,
+        date_time: Optional[dict] = None,
```

Comment:
Ahah yes I had come across it too, it was in my TODO thank you. As a 2nd step, I'd like to have 2 input parameters for all routers: departure_datetime and arrival_datetime instead, both of which are object datetime aware.

Reply:
It wasn't supported in Valhalla until ~ 2-3 months ago: https://github.com/valhalla/valhalla/pull/4071. 

I agree it'd be nicer to have a more generic way and have Valhalla convert that to its own request format. Until OTP there wasn't any OSS router supporting that, I guess that's why we didn't really think about it.